packages/terminal-extension/style/base.css

```diff
@@ -0,0 +1,5 @@
+/* Hide the stub toolbar that appears above terminals */
+
+.jp-Toolbar-micro {
```

Comment:
This selector should probably take into account whether the toolbar has items in it or not.

Reply:
Actually maybe this could be implemented in JupyterLab directly:

![image](https://user-images.githubusercontent.com/591645/227136545-c48e34d8-d8f2-4027-b142-4e039045e2aa.png)

Although in JupyterLab is looks like strange because there is the dock panel.

Reply:
@jtpio Per the code below, `jp-Toolbar-micro` is _only_ applied when a toolbar has no visible widgets.

https://github.com/jupyterlab/jupyterlab/blob/a3a0e3e7abf75e53810ed7ff8037c439cbb07697/packages/ui-components/src/components/toolbar.tsx#L58-L64

Reply:
Ah that's good to know. Then maybe this simple CSS change will just work and be enough.

Reply:
We could also put that under UI tests (either here or in JupyterLab) to make sure we don't hide the toolbar in case there is a widget in it.